cmd/minishift/cmd/config/config.go

```diff
@@ -97,6 +97,30 @@ var (
 	HostFoldersAutoMount = createFlag("hostfolders-automount", SetBool, nil, nil, true)
 
 	ImageCaching = createFlag("image-caching", SetBool, nil, nil, true)
+
+	// Preflight checks (before start)
+	SkipCheckKVMDriver    = createFlag("skip-check-kvm-driver", SetBool, nil, nil, true)
+	WarnCheckKVMDriver    = createFlag("warn-check-kvm-driver", SetBool, nil, nil, true)
```

Comment:
We really should rename _createFlag_ to _createConfigSetting_. The name implies that it creates a CLI flag somewhere, but this is not the case anymore. This is a list of configurable settings some of which are exposed as flags.